themes/src/main/resources/theme/keycloak.v2/login/theme.properties

```
@@ -87,4 +87,7 @@ kcLoginOTPListItemHeaderClass=pf-v5-c-tile__header pf-m-stacked
 kcLoginOTPListItemIconBodyClass=pf-v5-c-tile__icon
 kcLoginOTPListItemIconClass=fa fa-mobile
 kcLoginOTPListItemTitleClass=pf-v5-c-tile__title
-kcLoginOTPListSelectedClass=pf-m-selected
\ No newline at end of file
+kcLoginOTPListSelectedClass=pf-m-selected
+
+kcDarkModeClass=pf-v5-theme-dark
+kcDarkModeSwitchEnabled=true
```

For the regular theme properties we don't have to add any prefixes. In this case I'd say we can make this:
```suggestion
darkModeEnabled=true
```

---

+1

---

But wouldn't it all work and make code simpler if we just added this to the properties file?
```
# Comment out this line to disable dark mode
kcDarkModeClass=pf-v5-theme-dark
```
Then if `kcDarkModeClass` is null or empty then dark mode never gets added.

---

Yes that would also work, but @jonkoops wants the whole script removed when darkmode is not enabled

---

Resolving this as the PR was approved by all.